File: ApiExamples/VisualBasic/TestData/NumericDataSource.vb
Imports System

Namespace ApiExamples.TestData
	Public Class NumericDataSource
		Public Sub New(ByVal value1 As Integer, ByVal value2 As Double, ByVal value3 As Integer, ByVal [date] As Date)
			Me.Value1 = value1
			Me.Value2 = value2
			Me.Value3 = value3
			Me.Date = [date]
		End Sub

		Public Property Value1() As Integer

		Public Property Value2() As Double

		Public Property Value3() As Integer

		Public Property [Date]() As Date
	End Class
End Namespace

File: ApiExamples/CSharp/TestData/NumericDataSource.cs
using System;

namespace ApiExamples.TestData
{
    public class NumericDataSource
    {
        public NumericDataSource(int value1, double value2, int value3)
        {
            Value1 = value1;
            Value2 = value2;
            Value3 = value3;
        }

        public int Value1 { get; set; }
        public double Value2 { get; set; }
        public int Value3 { get; set; }
    }
}
